static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -31,6 +31,7 @@ const forcedAdFreeMode: boolean = !!window.location.hash.match(
 
 const userHasData = (): boolean => {
```

Comment:
you don't need to update `deleteOldData`?

Reply:
ironically it was already there https://github.com/guardian/frontend/blob/a8a29a996cecd094bae73ba0117fcef49fe3200b/static/src/javascripts/projects/common/modules/commercial/user-features.js#L94

Reply:
ha!

Reply:
could we factor out the list of relevant cookies and reference in both, to avoid similar issues in the future?

Reply:
seems like a good idea to me. @JustinPinner thoughts? (considering your previous point)

Reply:
Sounds sensible to me too